### PR TITLE
Postdeactivate PR suggestions

### PR DESCRIPTION
**Pull request by Dan503**
Made some suggestions for `index.html` and made some small code tweaks

<details>
<summary>PR Checklist</summary>
<br/>

__Please leave this checklist in your PR.__

- Source changes maintain stated browser compatibility.
- Includes updated `/docs/demo-bundle.js` if source or docs code was changed.
- Issue being fixed is referenced.
- Unit test coverage added/updated.
- E2E test coverage added/updated.
- Typings added/updated.
- README updated (API changes, instructions, etc.).
- Changes to dependencies explained.
- Changeset added (run `yarn changeset` locally to add one, and follow the prompts).
  - EXCEPTION: A Changeset is not required if the change does not affect any of the source files that produce the package bundle. For example, tooling changes, test updates, or a new dev-only dependency to run tests more efficiently should not have a Changeset since it will not affect package consumers.

</details>

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -76,14 +76,15 @@ <h2 id="animated-dialog-heading">animated dialog</h2>
...
-        message appears next to the &quot;returnFocus&quot; option. Note that the fade animation needs to be written yourself, it is not
-        built into the library.
-      </p>
+      <p>The activation button for this focus trap will fade out when the trap activates and fade in when the trap deactivates.</p>
+
+      <p>If the <code>returnFocus</code> option is <code>true</code>, the activation button will receive focus after the trap deactivates.</p>
+
+      <p>If the <code>returnFocus</code> option is <code>false</code>, the Deactivate button will retain focus after deactivation because that is the last element that was clicked.</p>
+
+      <p>Prior to deactivation. Once the trap has been deactivated (i.e. <code>onPostDeactivate</code>), a "trap deactivated" message appears next to the "returnFocus" option.</p>
```
Comment (stefcameron) [inline docs/index.html:85]: Did you mean to include, "Prior to deactivation." here? Doesn't seem to make sense in the flow of the text.

File: docs/js/animated-trigger.js [hunk 2]
```diff
@@ -16,6 +19,7 @@ const focusTrap = createFocusTrap('#animated-trigger', {
   onDeactivate: () => {
     container.classList.remove('is-active');
     trigger.classList.remove('is-triggered');
+    deactivatedFlag.classList.remove('is-hidden');
```
Comment (stefcameron) [inline docs/js/animated-trigger.js:22]: Wondering why you added this here. It's already in onPostDeactivate, which is where we previously had the console message.

**Review by stefcameron — CHANGES_REQUESTED** [review]
Thanks for the suggestions. Couple of questions for you...

File: docs/js/animated-trigger.js (hunk 2 above)
Comment (Dan503) [reply docs/js/animated-trigger.js:22]: Oh, I didn't see that...

Ok yeah sorry that isn't needed.

File: docs/index.html (hunk 1 above)
Comment (Dan503) [reply docs/index.html:85]: Oh, that was part of the previous sentence before. 

I don't think it's necessary. Can be removed.

**Review by Dan503 — COMMENTED** [review]
Sorry, yeah those were mistakes

**Review by stefcameron — APPROVED** [review]
I'm good with this if you are.

**Comment by stefcameron** [comment]
Well, I'll just merge it into my PR, and you can give your approval over there.

**Pull request merged**